### ISSUE TITLE
Copyright notice from HomeSeer: Please take this repo down 

### ISSUE THREAD

**Issue by jldubz** [alexdresko/HomeSeerNuget#3]
Hello,

This content is currently copyrighted material from HomeSeer which has been distributed without permission.  This material is not permitted to be redistributed in this manner, and we would appreciate it if you would take this repository down as soon as possible.

We are publishing new content to GitHub and NuGet and would like to avoid confusing anyone about what is official and what is not.

Thank you.

Regards,

The HomeSeer team.

**Comment by alexdresko** [comment]
You're making me take down my nuget package so you can post your own nuget package of the same thing? I get that, and you should definitely be the one to be posting a nuget package (finally), but to claim a copyright notice is laughable. 

I'll take this down as soon as I'm able, but don't expect it soon. You should have planned your rollout better.

**Comment by jldubz** [comment]
It is not the same thing.  The new content being published is for a brand new SDK.

In addition, this is a copywrite infringement according to the fact that you are redistributing material produced by HomeSeer that was not licensed as open-source to begin with.

You are sharing the Scheduler.DLL and HomeSeerAPI.DLL files which are proprietary packages produced by HomeSeer explicitly for distribution with the HomeSeer platform.

There is no need to be rude about this.

**Comment by alexdresko** [comment]
> you are redistributing material produced by HomeSeer that was not licensed as open-source to begin with.

I didn't open source anything. I'm simply making it more convenient to consume the assemblies that you provide publicly on your website. I made note changes to your assemblies. I simply made it easier for developers to use the same assemblies that you've been providing publicly for years. 

If I remove this package, how will HS3 developers obtain their assemblies? Will you also provide a HS3 nuget package to replace this one?

**Comment by jldubz** [comment]
No, we will not. HS3 plugin development is being deprecated now that we are beginning to publish content for HS4.

**Comment by alexdresko** [comment]
So if you're deprecating these assemblies, wouldn't it make sense just to rename this repo to `HomeSeer3Nuget`?  Be fair to the people, brother.

**Comment by alexdresko** [comment]
I can even transfer ownership of this repo to you. I don't mind. I just thinks it's wrong to be all "take it down, 'cause I'm a dick", especially without giving thanks for what I've done to make HS3 developer lives easier. 

I'm a software developer. I make decisions like a software developer.

**Comment by jldubz** [comment]
The initial request has been in the interest of consolidating public resources for plugin developers to avoid any confusion about what they should be using to develop plugins moving forward.

The content contained within this package is not even up to date.  The last commit was in October of 2018.  This does not tell me that this is a regularly maintained package.

You have done nothing but be rude to me in response as if I am personally attacking you, and this is not productive in any way.

**Comment by alexdresko** [comment]
> You have done nothing but be rude to me in response as if I am personally attacking you, and this is not productive in any way.

I am sorry. Please understand that I am emotionally involved with this big block of code that you're telling me to remove from the face of the earth. 

> The content contained within this package is not even up to date. The last commit was in October of 2018. This does not tell me that this is a regularly maintained package.

Again, that doesn't matter. 

> The initial request has been in the interest of consolidating public resources for plugin developers to avoid any confusion about what they should be using to develop plugins moving forward.

Again, I think we can prevent confusion without taking the repo down.

**Comment by alexdresko** [comment]
Edited for clarification:

> I am sorry. Please understand that I am emotionally involved with this big block of code that you're telling me to remove from the face of the earth.

**Comment by jldubz** [comment]
I believe we are on the same page but started off on the wrong foot, and I sincerely apologize for making you feel attacked or undervalued.

Please take a moment and breathe. I completely understand being emotionally connected to your work. How can you pour hours into something and NOT feel this way? I would be lying if I said I have never reacted like this.

The published content for the resources in question looks old and out-of-date, and a request to remove such content does not seem entirely out of the question to me.  This does not appear to be the case, though, given your feedback.

If you strongly feel like this should continue to be maintained, let's talk about it.  The HS3 plugin development API will be deprecated in the coming months now that an HS4 alpha has been published for plugin development; which includes a brand new plugin development SDK.  Updates to HS3 plugins will no longer be accepted at that point, but this is not the case until HS4 is officially released.  When HS4 is officially released, I still think this should be removed entirely from NuGet. It staying on GitHub is fine, but please note that it is deprecated and should not be used.

Currently, I will ask you to either clearly denote that this is for HS3 ONLY and update it with the most currently up-to-date DLLs, or remove it please and thank you.

**Comment by jldubz** [comment]
Adding in from the other repo: 

Please also properly link to related source material from HomeSeer, such as [https://homeseer.com/developer-support/](https://homeseer.com/developer-support/) which links to all of the resources a developer may need to get going with HS3 plugin development.  We would also appreciate it if you provided a link from your readme to the [HomeSeer website](https://homeseer.com/) so people know what platform this is for and see that you are extending content provided through HomeSeer.

**Comment by alexdresko** [comment]
> I believe we are on the same page but started off on the wrong foot, and I sincerely apologize for making you feel attacked or undervalued.

I apologize as well. My wife would hit me for acting the way I did today. :) 

> Please take a moment and breathe. I completely understand being emotionally connected to your work. How can you pour hours into something and NOT feel this way? I would be lying if I said I have never reacted like this.

You don't often hear of "friendly take down requests", so I was immediately on guard when I saw this issue created. 

> The published content for the resources in question looks old and out-of-date, and a request to remove such content does not seem entirely out of the question to me. This does not appear to be the case, though, given your feedback.
> 
> If you strongly feel like this should continue to be maintained, let's talk about it. The HS3 plugin development API will be deprecated in the coming months now that an HS4 alpha has been published for plugin development; which includes a brand new plugin development SDK. Updates to HS3 plugins will no longer be accepted at that point, but this is not the case until HS4 is officially released. When HS4 is officially released, I still think this should be removed entirely from NuGet. It staying on GitHub is fine, but please note that it is deprecated and should not be used.

I will attempt to rename the nuget package to reflect the fact that it's for HS3, but I do not think it should be taken down. There are developers who depend on this package and always will as long as HS3 runs on their system. The plugin store doesn't matter to me. I have custom plugins running in HS3 that I need to maintain, and I'm probably not the only one. And you can't force everyone to upgrade to HS4.

I will comply with these requests over the weekend:

*  > clearly denote that this is for HS3 development ONLY
* > properly link to related source material from HomeSeer, such as https://homeseer.com/developer-support/ which links to all of the resources a developer may need to get going with HS3 plugin development
* > [provide] a link from your readme to the HomeSeer website so people know what platform this is for and see that you are extending content provided through HomeSeer.
* I will try to rename the nuget package to reflect the fact that it's for HS3

**Comment by jldubz** [comment]
Thank you very much.  I look forward to working with you on this in the future.
